packages/spartan/extra/gen_resolvconf.py

```diff
@@ -69,7 +69,20 @@ def check_server(addr):
 
 # If Spartan is not up, fall back, and insert the upstreams
 else:
-    fallback_servers = os.environ['RESOLVERS'].split(',')
+    fallback_servers = []
+
+    # Resolvconf does not support custom ports, skip if not default
+    for ns in os.environ['RESOLVERS'].split(','):
+        ip, separator, port = ns.rpartition(':')
+        if not separator:
+            fallback_servers.append(ns)
+            continue
+        if port == "53":
+            fallback_servers.append(ip)
+            continue
+        print('Skipping DNS server {}: non-default ports are not supported in resolvconf'.format(
```

Comment:
/etc/resolv.conf -- resolvconf is yet another piece of DNS configuration software